architecture/core_concepts/routes.adoc

```diff
@@ -201,12 +201,14 @@ variables on the *deployment config* for the router to alter its configuration.
 |`*NAMESPACE_LABELS*` |  | A label selector to apply to namespaces to watch, empty means all.
 |`*PROJECT_LABELS*` |  | A label selector to apply to projects to watch, emtpy means all.
 |`*RELOAD_SCRIPT*` |  | The path to the reload script to use to reload the router.
+|`*ROUTER_ALLOWED_DOMAINS*` | | A comma separated list of domains that the hostname in a route can only be part of.  Any subdomain in the domain can be used either.  Checked after denied domains.  If set, everything outside of the allowed domains will be rejected.
```

Comment:
s/either/too ?
s/Checked after denied domains/Option 'ROUTER_DENIED_DOMAINS' overrides any values given in this option/ ?